### PR TITLE
Update zeitschrift-fur-allgemeinmedizin.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/comment/281386/#Comment_281386

### CODE REVIEW

File: zeitschrift-fur-allgemeinmedizin.csl [hunk 1]
```diff
@@ -14,12 +14,13 @@
...
     <issn>1433-6251</issn>
     <eissn>1439-9229</eissn>
-    <updated>2017-07-25T14:09:51+00:00</updated>
+    <updated>2017-07-26T21:25:26+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="de">
     <terms>
       <term name="accessed">letzter Zugriff am</term>
+      <term name="et-al">et al</term>
```
Comment (rmzelle) [inline zeitschrift-fur-allgemeinmedizin.csl]: @POBrien333, I haven't checked the guidelines, but should this be "et al."? It's an abbreviation ("et alia"), so a period is almost always used.

Comment (POBrien333) [inline zeitschrift-fur-allgemeinmedizin.csl]: It actually has the suffix on the author macro, so that should've sufficed, no?

Comment (rmzelle) [inline zeitschrift-fur-allgemeinmedizin.csl]: Even then the term should have a period. (term definitions should be correct independent of the context in which they're used, and CSL processors generally suppress duplicated punctuation to deal with e.g. extra periods)

**Comment by rmzelle** [comment]
Thanks!

**Comment by POBrien333** [comment]
Ah, dang. Thanks for fixing that! :)

**Comment by rmzelle** [comment]
Nitpicking to the rescue!

image

**Pull request merged**